library/billing.inc

```diff
@@ -229,40 +253,56 @@ function updateClaim(
         "x12_partner_id = '$partner_id'";
         ****/
         sqlBeginTrans();
-        $version = sqlQuery("SELECT IFNULL(MAX(version),0) + 1 AS increment FROM claims WHERE patient_id = ? AND encounter_id = ?", array( $patient_id, $encounter_id ));
+        $sqlBindArray = array();
+        $sqlBindArray[] = $patient_id;
+        $sqlBindArray[] = $encounter_id;
+        $version = sqlQuery("SELECT IFNULL(MAX(version),0) + 1 AS increment FROM claims WHERE patient_id = ? AND encounter_id = ?", $sqlBindArray);
```

Comment:
This was not necessary. Only need to to $sqlBindArray when you don't know the full query.